### PR TITLE
creating worker requests

### PR DESCRIPTION
**Pull request by ankittkp**

### CODE REVIEW

File: cmd/workers/workers.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+var client = &http.Client{}
+var Wg sync.WaitGroup
+
+func worker(id int, itr int, ch chan<- string) {
+	defer Wg.Done()
+	for i := 0; i < itr; i++ {
+		randomWaitTime := time.Duration(rand.Int63n(int64(time.Second)))
+
+		time.Sleep(randomWaitTime)
+		req, err := http.NewRequest("GET", "http://localhost:8080/health", nil)
```
Comment (ankittkp) [inline cmd/workers/workers.go:20]: I called only 1 endpoint here, imagine calling all of our endpoints and millions of requests at 1 end point would be insane

File: cmd/workers/main.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	NoOfJobsPerWorker = 100000
+)
+
+func main() {
+	ch := make(chan string)
+	Wg.Add(NoOfWorkers)
+	go CloseWorkersChannel(ch)
+
+	for i := 0; i < NoOfWorkers; i++ {
+		go worker(i, NoOfJobsPerWorker, ch)
```
Comment (oo-weave) [inline cmd/workers/main.go:18]:
```suggestion
	go CloseWorkersChannel(ch)

	for i := 0; i < NoOfWorkers; i++ {
	        Wg.Add(1)
		go worker(i, NoOfJobsPerWorker, ch)
```
[icza](https://stackoverflow.com/a/65213822) says to add in the loop right before the goroutine. 🤷🏻 🤔

File: cmd/workers/workers.go (hunk 1 above)
Comment (oo-weave) [reply cmd/workers/workers.go:20]: 🤯  
#ddos machine 😆

File: cmd/workers/main.go (hunk 2 above)
Comment (ankittkp) [reply cmd/workers/main.go:18]: just saw that thats great, thanks

**Pull request merged**